jme3-core/src/main/resources/Common/ShaderLib/Instancing.glsllib

```diff
@@ -106,4 +106,4 @@ vec3 TransformWorldNormal(vec3 normal) {
 }
 
  
-#endif
\ No newline at end of file
+#endif
```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR